src/elm/State.elm

```diff
@@ -14,7 +14,7 @@ import Types exposing (..)
 
 initModel : Model
 initModel =
-    { route = LandingRoute
+    { route = FormRoute
```

Comment:
should probably try not to commit these changes

Reply:
oooophs

Reply:
done